frontend/src/utils/keyFrames.js

```diff
@@ -0,0 +1,60 @@
+import { keyframes } from '@emotion/react';
+
+export const fadeIn = keyframes`{
+    from {
+        opacity:0;
+    }
+  
+    to {
+        opacity:1;
+    }
+}`;
+
+export const fadeInLeft = keyframes`
+0% {
+    -webkit-transform: translate3d(-100%,0,0);
+    opacity: 0;
+    transform: translate3d(-100%,0,0)
+}
+
+to {
+    -webkit-transform: translateZ(0);
+    opacity: 1;
+    transform: translateZ(0)
+}`;
```

Comment:
```suggestion
from {
    -webkit-transform: translate3d(-100%,0,0);
    opacity: 0;
    transform: translate3d(-100%,0,0)
}

to {
    -webkit-transform: translateZ(0);
    opacity: 1;
    transform: translateZ(0)
}`;
```
Ngoài ra check lại indent của các animation. Có cái thì from thụt vô, có cái thì thụt ra